internal/api/user_devices.go

```diff
@@ -111,6 +112,26 @@ func (s *userDeviceService) ListUserDevicesForUser(ctx context.Context, req *pb.
 	return &pb.ListUserDevicesForUserResponse{UserDevices: out}, nil
 }
 
+func (s *userDeviceService) ListDevicesForWalletAddress(ctx context.Context, req *pb.ListDevicesForWalletAddressRequest) (*pb.ListDevicesForWalletAddressResponse, error) {
+	vNft, err := models.VehicleNFTS(
+		models.VehicleNFTWhere.OwnerAddress.EQ(null.BytesFrom(common.FromHex(req.UserAddress))),
+		qm.Load(models.VehicleNFTRels.UserDevice),
```

Comment:
We want integration information, as on line 99. I think we are missing that here.

Reply:
We also want NFT pairing information, as on line 98, we go into the autopi_units table.